centos/7/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM centos:7
+
+LABEL maintainer me@danvaida.com
+
+ENV container docker
+
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+    rm -f /lib/systemd/system/multi-user.target.wants/*;\
+    rm -f /etc/systemd/system/*.wants/*;\
+    rm -f /lib/systemd/system/local-fs.target.wants/*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+    rm -f /lib/systemd/system/basic.target.wants/*;\
+    rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+VOLUME [ "/sys/fs/cgroup" ]
+
+RUN echo 'root:root' | chpasswd
+
+RUN yum update -y && \
+    yum install -y epel-release
```

Comment:
not really. there's a new install command on line 22. or are you asking why am I not installing everything at once?